roles/timer/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Fail when timer_name is empty
+  fail: msg="timer_name is empty"
+  when: not timer_name
+
+- name: Upload service and timer
+  template:
+    src: "job.{{ item }}"
+    dest: /lib/systemd/system/{{ timer_name }}.{{ item }}
+  with_items:
+  - service
+  - timer
+  register: template_result
+
+- name: Reload systemd
+  command: systemctl daemon-reload
```

Comment:
please use the ansible module for this:
https://docs.ansible.com/ansible/latest/modules/systemd_module.html#parameter-daemon_reload

Reply:
fixed

Reply:
nope ...

Reply:
@mortzu you asked for ... this task is unnecessary